src/program/models.py

```diff
@@ -528,8 +579,8 @@ def serialize(self, user=None):
             'title': self.event.title,
             'slug': self.event.slug + '-' + str(self.id),
             'event_slug': self.event.slug,
-            'from': self.when.lower.astimezone().isoformat(),
-            'to': self.when.upper.astimezone().isoformat(),
+            'from': self.when.lower.astimezone().isoformat() if self.when else None,
+            'to': self.when.upper.astimezone().isoformat() if self.when else None,
```

Comment:
Why this? Pretty sure that the Elm app will die if it receives anything that isn't an ISO formatted datetime sting.

Reply:
This one was because I had real errors... `NoneType blah blah doesn't have property lower` kind of stuff... if you look at its definition, it is nullable!

Reply:
Hmm, strange. Haven't encountered that before myself :)